src/vcpkg/base/files.cpp

```diff
@@ -3001,6 +3001,47 @@ namespace vcpkg
                     return false;
                 }
 
+                if (options == CopyOptions::update_existing)
+                {
+                    WIN32_FILE_ATTRIBUTE_DATA attributes_destination;
+                    if (GetFileAttributesExW(wide_destination.c_str(), GetFileExInfoStandard, &attributes_destination))
+                    {
+                        WIN32_FILE_ATTRIBUTE_DATA attributes_source;
+                        if (!GetFileAttributesExW(wide_source.c_str(), GetFileExInfoStandard, &attributes_source))
+                        {
+                            ec.assign(GetLastError(), std::system_category());
+                            return false;
+                        }
+
+                        // Do not copy if destination file is equal or more recent than source file
+                        if (CompareFileTime(&attributes_destination.ftLastWriteTime,
+                                            &attributes_source.ftLastWriteTime) >= 0)
+                        {
+                            ec.clear();
+                            return false;
+                        }
+
+                        // Overwrite file because a newer version was found
+                        if (::CopyFileW(wide_source.c_str(), wide_destination.c_str(), FALSE))
+                        {
+                            ec.clear();
+                            return true;
+                        }
+                        else
+                        {
+                            last_error = GetLastError();
+                            ec.assign(static_cast<int>(last_error), std::system_category());
+                            return false;
+                        }
```

Comment:
```suggestion
                            last_error = GetLastError();
                            ec.assign(static_cast<int>(last_error), std::system_category());
                            return false;
```

else implied by return at the end of the if block.